pkg/http/proxy_test.go

```diff
@@ -76,7 +76,7 @@ func TestNewHeaderPruningProxy(t *testing.T) {
 
 			resp := httptest.NewRecorder()
 			req := httptest.NewRequest(http.MethodPost, test.url, nil)
-			req.Header = test.header
+			req.Header = test.headers
```

Comment:
this makes me feel like the original might've been right, I think. There are multiple headers, but the actual field in `req` that the test case is setting (and the type of the http.Header map containing multiple headers) is - whether that was a good decision on upstream go's part or not 😅 - singular?

Reply:
hm, dunno. :) feels wrong
But I don't care strongly.

Reply:
yah, agree, but.... blame go ;p